streaming/index.js

```diff
@@ -622,40 +622,61 @@ const startServer = async () => {
 
     log.verbose(req.requestId, `Starting stream from ${ids.join(', ')} for ${accountId}`);
 
-    // Currently message is of type string, soon it'll be Record<string, any>
-    const listener = message => {
-      const { event, payload, queued_at } = message;
+    const transmit = (event, raw_payload, filter_results) => {
+      let payload = raw_payload;
+      if ((event === 'update' || event === 'status.update') && filter_results && filter_results.length > 0) {
+        // copy the payload and add the `filtered` attribute, an Array of FilterResult
+        // https://docs.joinmastodon.org/entities/Status/#filtered
+        payload = {
+          ...payload,
+          filtered: filter_results
+        };
+      }
 
-      const transmit = (payload) => {
-        const now = new Date().getTime();
-        const delta = now - queued_at;
```

Comment:
@ClearlyClaire I ended up dropping the `queued_at` / Delay logging from here as it was only present in the `silly` log level and wasn't available for majority of events (as only a small fraction actually contain a queued_at timestamp, and it's unclear as to exactly what this is measuring as a metric.

Reply:
Further, the `queued_at` value is removed entirely by https://github.com/mastodon/mastodon/pull/26173